src/04-modality-specific-files/01-magnetic-resonance-imaging-data.md

```diff
@@ -74,10 +74,11 @@ first "effective" echo and the center of the last "effective" echo.
 
 #### RF & Contrast
 
-| Field name                  | Definition                                                                                                            |
-| :-------------------------- | :-------------------------------------------------------------------------------------------------------------------- |
-| FlipAngle                   | RECOMMENDED. Flip angle for the acquisition, specified in degrees. Corresponds to: DICOM Tag 0018, 1314 `Flip Angle`. |
-| MultibandAccelerationFactor | RECOMMENDED. The multiband factor, for multiband acquisitions.                                                        |
+| Field name                  | Definition                                                                                                                                                                                                                                                                                                                                       |
+| :---------------------------- | :--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| FlipAngle                   | RECOMMENDED. Flip angle for the acquisition, specified in degrees. Corresponds to: DICOM Tag 0018, 1314 `Flip Angle`.                                                                                                                                                                                                                            |
+| MultibandAccelerationFactor | RECOMMENDED. The multiband factor, for multiband acquisitions.                                                                                                                                                                                                                                                                                   |
+| NegativeContrast           | OPTIONAL. Boolean ("true" or "false") value specifying whether increasing voxel intensity (within sample voxels) denotes a decreased value with respect to the contrast suffix. This is commonly the case when Cerebral Blood Volume is estimated via usage of a contrast agent in conjunction with a T2\* weighted acquisition protocol.        |
```

Comment:
```suggestion
| NegativeContrast           | OPTIONAL. Boolean (`true` or `false`) value specifying whether increasing voxel intensity (within sample voxels) denotes a decreased value with respect to the contrast suffix. This is commonly the case when Cerebral Blood Volume is estimated via usage of a contrast agent in conjunction with a T2\* weighted acquisition protocol.        |
```

Reply:
I saw that this was also an alternative in the current document, but "true" and "false" here are quoted strings (like what you normally put in quotation marks in all sort of text), and not inlined code.

Reply:
I want to avoid giving the impression those values are strings not booleans in strict JSON sense.

Reply:
ok